compute/admin_guide/vulnerability_management/detect_vulns_unpackaged_software.adoc

```diff
@@ -15,6 +15,9 @@ The following list shows the package types that are currently supported and can
 
 Prisma Cloud uses a variety of approaches for package detection, these are purpose-built differently for images and hosts. For example, the host Defender only scans applications, as well as language-based packages, if the processes are running. The Windows Defender only scans packages that are installed with a package manager, missing Microsoft hotfixes, and .net framework applications.
 
+NOTE: For an application that originates from an OS package, the vulnerability data for CVEs are sourced from the relevant feed for the OS package. In some cases, like with Amazon Linux and Photon OS, this CVE information is provided in security advisories such as Amazon Linux Security Advisories (ALAS) for Amazon, and PHSA for Photon. In such cases, the correlation for the relevant vulnerabilities is limited.
```

Comment:
This is in the RN.

Do we need it in the Admin Guide @divyabhushan?

Reply:
Yes, as discussed on the ticket. @Pubs-MV